docs/tutorials/t1.rst

```diff
@@ -102,6 +108,62 @@ using ``child_data``
         for result in sub_data.analysis_results():
             print(result)
 
+:math:`T_1` experiments with kerneled measurement
+---------------------------------------------------
+:math:`T_1` experiment can also be done with kerneled measurement.
+If we will set the running option `meas_level=MeasLevel.KERNELED`, the
+job will not discriminate the data and will not label it. In T1 experiment,
+as we know that in :math:`P(1|t=0)=1`, we will add measurement for delay=0
+and for extreme delay. In this configuration we know that the data starts from
+a point [I,Q] that is close to a logical value '1' and ends at a point [I,Q]
+that is close to a logical value '0'. Beside that, We will need to use analysis
+option to normalize the output.
```

Comment:
```suggestion
that is close to a logical value '0'. 
```

The analysis option is added automatically.

Reply:
Changed as suggested.